style/pileup.css

```diff
@@ -4,11 +4,47 @@
  * You are encouraged to override anything you like.
  */
 
+.pileup-root {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.pileup-root > .track {
+  display: flex;
+  flex-direction: row;
+}
+.track-label {
+  flex: 0 0 100px;  /* fixed-width track labels */
+  text-align: right;
+  font-family: Helvetica Neue, Helvetica, Arial, sans;
```

Comment:
Nice catch! I quoted correctly & changed `sans` to `sans-serif`.

I'd prefer to keep TypeKit out of core pileup.js. But users of pileup (like Cycledash) are free to use it in their own stylesheets.